tsconfig.json

```diff
@@ -207,14 +207,23 @@
       "@dynatrace/shared/barista-definitions": [
         "libs/shared/barista-definitions/src/index.ts"
       ],
-      "@environments/*": ["environments/*.ts"],
+      "@dynatrace/shared/node": ["libs/shared/node/src/index.ts"],
       "@dynatrace/shared/design-tokens": [
         "libs/shared/design-tokens/src/index.ts"
       ],
       "@dynatrace/fluid-design-tokens": [
         "libs/shared/design-tokens/generated/index.ts"
       ],
-      "@dynatrace/shared/node": ["libs/shared/node/src/index.ts"]
+      "@dynatrace/fluid-design-tokens/meta/aliases/*": [
+        "libs/shared/design-tokens/generated/aliases/*.json"
+      ],
+      "@dynatrace/fluid-design-tokens/meta/global/*": [
+        "libs/shared/design-tokens/generated/global/*.json"
+      ],
+      "@dynatrace/fluid-design-tokens/meta/patterns/*": [
+        "libs/shared/design-tokens/generated/patterns/*.json"
+      ],
```

Comment:
If there's a better way to do this, please let me know.

Reply:
Currently ther is no better solution I think but we should be aware that nesting does not work.

Reply:
depends on how the globbing works, you could try: 
`"@dynatrace/fluid-design-tokens/meta/{aliases,global,patterns}/*"`
But this is an untested guess.

Reply:
FYI this is a glob for webpack :)

Reply:
> depends on how the globbing works, you could try:
> `"@dynatrace/fluid-design-tokens/meta/{aliases,global,patterns}/*"`
> But this is an untested guess.

Doesn't work, unfortunately.